CSC8503/InventoryBuffSystem/PlayerInventory.cpp

```diff
@@ -0,0 +1,93 @@
+	#include "PlayerInventory.h"
+
+	using namespace InventoryBuffSystem;
+
+	void PlayerInventory::Init()
+	{
+		for (int playerNo = 0; playerNo < 4; playerNo++)
+		{
+			for (int invSlot = 0; invSlot < 2; invSlot++)
```

Comment:
Same for this, you've even used it further down as MAX_INVENTORY_SLOTS you just need to add it here